pulser-simulation/pulser_simulation/hamiltonian.py

```diff
@@ -157,6 +157,7 @@ def _build_collapse_operators(
         if "eff_noise" in config.noise_types:
             for id, rate in enumerate(config.eff_noise_rates):
                 op = np.array(config.eff_noise_opers[id])
+                print(op)
```

Comment:
leftover debug print

Reply:
Arg damn, I didn't even used it for debug 😢